rest/client/methods.go

```diff
@@ -1571,3 +1571,30 @@ func (c *communicatorImpl) CompareTasks(ctx context.Context, tasks []string, use
 
 	return results.Order, results.Logic, nil
 }
+
+// FindHostByIpAddress queries the database for the host with ip matching the ip address
+func (c *communicatorImpl) FindHostByIpAddress(ctx context.Context, ip string) (*model.APIHost, error) {
+	info := requestInfo{
+		method: http.MethodGet,
+		path:   fmt.Sprintf("hosts/ip_address/%s", ip),
+	}
+
+	resp, err := c.request(ctx, info, nil)
+	if err != nil {
+		err = errors.Wrapf(err, "error sending request to find host by ip address")
```

Comment:
We don't need to assign this to the error, we can just return it explicitly

Reply:
done